contrib/dtach/template.py

```diff
@@ -0,0 +1,18 @@
+pkgname = "dtach"
+pkgver = "0.9"
+pkgrel = 0
+build_style = "configure"
```

Comment:
```suggestion
build_style = "gnu_configure"
```

Reply:
this also needs automake in hostmakedepends and then might fail to regen given the age, but it is autotools so this is the correct style

if you can't figure out the regen then put configure_gen = [] too

Reply:
looks like it does regen for now at least though a few warnings about deprecated stuff are logged.